apps/music/locales/music.en-US.properties

```diff
@@ -51,3 +51,15 @@ overlay-cancel-button = Cancel
 music-paused-notification = Music paused during the call
 done = Done
 
+# The following strings are spoken by screen readers and not shown on the
+# screen.
+
+share-song.ariaLabel = share
+player-album-rating.ariaLabel = Rating
+rating-star-1.ariaLabel = 1 star
```

Comment:
With the comment in the html file we'll have this for 'rating-star' entry:
rating-star={[ plural(n) ]}
rating-star[zero]  = {{n}} stars
rating-star[one]   = {{n}} star
rating-star[two] = {{n}} stars
rating-star[few] = {{n}} stars
rating-star[many] = {{n}} stars
rating-star[other] = {{n}} stars